CMakeLists.txt

```diff
@@ -113,7 +113,9 @@ find_package(BZip2)
 set(CMAKE_THREAD_PREFER_PTHREAD)
 set(THREADS_PREFER_PTHREAD_FLAG)
 set(THREADS_USE_PTHREADS_WIN32 true)
+if(NOT MINGW)
```

Comment:
don't we also need Threads with mingw?

Reply:
@mpreisler if you skip this threads check when using mingw, the build completes successfully.

Reply:
ok, but why? where does it get pthreads?

Reply:
I have already commented on https://github.com/OpenSCAP/openscap/issues/1073. I think that would work only if you build OpenSCAP without probes and probe subsystem.

I think we need to look into cmake/FindThreads.cmake file instead.
There is a variable THREADS_USE_PTHREADS_WIN32, which is used in root CMakeLists.txt around line 115.

Try something like:

if(NOT MINGW)
   set(THREADS_USE_PTHREADS_WIN32 true)
endif()

If that does not help, we will most likely need to dive into cmake/FindThreads.cmake .